src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@import 'variables';
+@import 'header';
+@import 'nav';
+@import 'catalog';
+@import 'card';
+@import 'star';
+
+* {
```

Comment:
The universal selector (*) is used here, which can be very performance-intensive as it applies to all elements. Consider using more specific selectors if possible.